### PR TITLE
Fix #187 : Update Oppiabot comment for asking team-leads to merge other's PRs. 

### LINKED ISSUE #187: Message asking project lead to merge the PR should have info on why they are expected to merge other's PR
**Current situation:**
When the author doesn't have the rights to merge a PR then oppiaBot asks the team lead to merge the PR:
```
Hi @<team-lead>, this PR is ready to be merged. Before you merge it, please make sure that there are no pending comments that require action from the author's end. Thanks!
```

**Suggestion:**

In general, we expect PR authors to get their PRs merged but if in any case, we are asking anyone else to merge the PR then the bot should add info about why they are asked to merge the PR.

### PR DESCRIPTION
**Pull request by HarshKhilawala**
## Explanation

PR fixes #187 
Modifies the message for team leads to merge the PR for the author with no write permissions to oppia. 

From : 
```
Hi @<team-lead>, this PR is ready to be merged. Before you merge it, please make sure that there are no pending comments that require action from the author's end. Thanks!
```

To: 
```
Hi @<team-lead>, this PR is ready to be merged. Author of this PR do not have permissions to merge this PR. Before you merge it, please make sure that there are no pending comments that require action from the author's end. Thanks!
```

## Checklist
- [x] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [x] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

**Comment by HarshKhilawala** [comment]
@vojtechjelinek @jameesjohn  PTAL!

File: lib/checkPullRequestReview.js [hunk 1]
```diff
@@ -127,8 +127,9 @@ const handleApprovalByAllReviewers = async (context, pullRequest) => {
     const commentBody =
       'Hi @' +
       projectOwner +
-      ', this PR is ready to be merged. Before you merge it, please ' +
-      'make sure that there are no pending comments that require ' +
+      ', this PR is ready to be merged. Author of this PR ' +
+      'do not have permissions to merge this PR. Before you merge it, ' +
```
Comment (vojtechjelinek) [inline lib/checkPullRequestReview.js:131]:
```suggestion
      'does not have permissions to merge this PR. Before you merge it, ' +
```

File: lib/checkPullRequestReview.js [hunk 2]
```diff
@@ -127,8 +127,9 @@ const handleApprovalByAllReviewers = async (context, pullRequest) => {
...
+      'please make sure that there are no pending comments that require ' +
       "action from the author's end. Thanks!";
```
Comment (vojtechjelinek) [inline lib/checkPullRequestReview.js:133]: Could you change this to
```suggestion
      'action from the author's end. Thanks!';
```

**Review by vojtechjelinek — CHANGES_REQUESTED** [review]
Thanks! I have two small comments.

**Comment by HarshKhilawala** [comment]
PTAL! @vojtechjelinek

File: lib/checkPullRequestReview.js [hunk 3]
```diff
@@ -127,9 +127,10 @@ const handleApprovalByAllReviewers = async (context, pullRequest) => {
     const commentBody =
       'Hi @' +
       projectOwner +
-      ', this PR is ready to be merged. Before you merge it, please ' +
-      'make sure that there are no pending comments that require ' +
-      "action from the author's end. Thanks!";
+      ', this PR is ready to be merged. Author of this PR ' +
+      'does not have permissions to merge this PR. Before you merge it, ' +
+      'please make sure that there are no pending comments that require ' +
+      'action from the author's end. Thanks!';
```
Comment (HarshKhilawala) [inline lib/checkPullRequestReview.js:133]: @vojtechjelinek In this line, only double quote can be used as "author's" has single quote which will end the statement.

**Comment by HarshKhilawala** [comment]
@vojtechjelinek PTAL!

**Review by vojtechjelinek — APPROVED** [review]
LGTM!

**Pull request merged**